applications/FluidDynamicsApplication/custom_elements/symbolic_stokes.cpp

```diff
@@ -0,0 +1,4943 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main author:     Daniel Diez

```

Comment:
Not me : P

Reply:
Is it fine now? If not, please suggest changes. 😄

Reply:
NA